src/julia/install.jl

```diff
@@ -64,7 +64,7 @@ catch err
 end
 
 
-ENV["PYTHON"] = python
+ENV["PYTHON"] = lowercase(python)
```

Comment:
`python` should be normalized on the Python side. Also, I think using `lowercase` would break for non-Windows users.

Also, why is this necessary? Please see https://github.com/JuliaPy/pyjulia/issues/67#issuecomment-480649375

Reply:
Ah! Yep, totally forgot on that. Thanks for pointing out.